packages/utils/src/index.ts

```diff
@@ -47,7 +47,6 @@ export type {
   Constructable,
   DeepMerge,
   MergeInsertions,
-  Nullable,
```

Comment:
This is an internal package, TypeScript doesn't provide these types in user projects